lpcompiler.pas

```diff
@@ -315,6 +320,12 @@ procedure TLapeCompiler.setTokenizersPeek(Peek: Boolean);
       __LapeTokenizerBase(FTokenizers[i]).FInPeek := Peek;
 end;
 
+procedure TLapeCompiler.HandleHint(Msg: lpString; ADocPos: TDocPos);
```

Comment:
Is it maybe an idea to use an enum to define the type of hint, and then an `array of const` parameter list to fill in the arguments. For example, `HandleHint(lchUnusedVariable, [VariableName])`.